src/main/java/frc/subsystem/CargoTracker.java

```diff
@@ -0,0 +1,201 @@
+package frc.subsystem;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.networktables.EntryListenerFlags;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.Constants;
+import frc.utility.cargotracking.CargoPairing;
+import frc.utility.cargotracking.CargoPosition;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.joml.AxisAngle4d;
+import org.joml.Vector2d;
+import org.joml.Vector3d;
+
+import java.util.*;
+
+import static frc.robot.Constants.*;
+
+public class CargoTracker extends AbstractSubsystem {
+
+    public CargoTracker() {
+        super(1000);
+    }
+
+    @Override
+    public void selfTest() {
+
+    }
+
+    @Override
+    public void logData() {
+
+    }
+
+    @Override
+    public void close() throws Exception {
+
+    }
+
+    @NotNull NetworkTableEntry blueCargoTable = NetworkTableInstance.getDefault().getEntry("/realsense/pos_blue");
```

Comment:
can you leave a comment about what the structure of this should be and what the contents are?

Reply:
added in bd2e50f63269369232d1a08cedd91a5b87c9278c